### PR TITLE
Fix for #404

### LINKED ISSUE #404: [Merge PDF] Allow users to merge multiple files
- Remove the select first PDF & select second PDF option from `Merge Files`
- Move Merge Files button to top
- Add a `Select PDF` button below it
- After the file is selected, add it to the list (below the Merge Files button)

- In the bottom sheet also, remove the dialog & just keep adding PDF to the list (when user clicks it), and just show a snackbar `PDF has been added to the list`

### PR DESCRIPTION
**Pull request by vyankatesh24**
# Description
Added an option to add multiple files for merging in Merge PDF Fragment.
image

image

image

Fixes #404 

## Type of change
Just put an x in the [] which are valid.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes.
- [x] `./gradlew assembleDebug assembleRelease`
- [x] `./gradlew checkstyle`

### CODE REVIEW

**Comment by Swati4star** [comment]
Instead of dialog, add a cross button to the right end of listview only.

File: app/src/main/java/swati4star/createpdf/fragment/MergeFilesFragment.java [hunk 1]
```diff
@@ -112,19 +117,14 @@ void onViewFilesClick(View view) {
         }
     }
 
-    @OnClick({R.id.fileonebtn, R.id.filetwobtn})
+    @OnClick({R.id.selectFiles})
```
Comment (Swati4star) [inline app/src/main/java/swati4star/createpdf/fragment/MergeFilesFragment.java]: Remove the curly brackets

**Comment by vyankatesh24** [comment]
https://github.com/Swati4star/Images-to-PDF/pull/411#issuecomment-414934785

We can add more options there like `Open pdf` etc.

**Comment by vyankatesh24** [comment]
Shall I add `Open Pdf` option or add `cross button` or both.

**Comment by Swati4star** [comment]
Both.

**Comment by Swati4star** [comment]
Post the new screenshot also.

**Comment by vyankatesh24** [comment]
image

**Pull request merged**